querybook/webapp/const/search.ts

```diff
@@ -39,7 +39,7 @@ export interface ISearchTableParams {
     filters?: Array<[filterName: string, filterValue: any]>;
     fields?: string[];
     sort_key?: string | string[];
-    sort_order?: 'desc' | 'asc';
+    sort_order?: 'desc' | 'asc' | Array<[desc: string, asc: string]>;
```

Comment:
should typing be `Array<'desc' | 'asc'>`

Reply:
You're right!